version.txt

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
To work with the `jenkins.sh` this file should be called `VERSION.txt`, which is what we use elsewhere